CONTRIBUTING.md

```diff
@@ -137,18 +137,21 @@ If we accept the PR, a [maintainer](MAINTAINERS.md) will merge your change and u
 
 If we reject the PR, we will close the pull request with a comment explaining why. This decision isn't always final: if you feel we have misunderstood your intended change or otherwise think that we should reconsider then please continue the conversation with a comment on the PR and we'll do our best to address any further points you raise.
 
-## Feature Branches
+## Using Feature Branches
 
-For long running features that need collaboration between multiple members of the community, we use feature branches. We use a hybrid of the feature branch development model and the continuous integration development model. Maintainers of a repository can create a feature branch, a corresponding label associated with the feature and a meta issue associated with the feature. Each feature branch will have the following safeguards:
+Our recommend approach for development is doing frequent small PR merges to main. This lets us catch integration issues earlier and makes the development visible to everyone.  It's okay if it's not the complete feature, as long as the PR won’t break a build or any existing functionality at least. 
 
-1. High frequency integration: PR's to `main` are raised not once the feature is complete, but whenever we reach the closest integration point during development. For example, once the plugin is bootstrapped, or a core feature is complete. We also frequently rebase changes from main back into the feature branch.
-    1. This makes the integration frequency much higher and allows us to catch integration issues much quicker.
-    2. This still lets collaboratively develop on a big feature that is not ready for main.
-2. Treat feature branch PR's the same as PR's to `main`. CI should run on all PR's, no incomplete work should be merged, tests are necessary, etc.
-   1. This maintains the code quality going into each feature making the integration to main PR's much easier and quicker.
-   2. More visibility during development since it gives reviewers the necessary time to review each PR.
-3. Feature specific labels: This helps identify feature related issues and PR's.
+But sometimes it may be useful to create a feature branch.  This allows you work on long-running disruptive features in isolation.   The reason we don't recommend it is because it still requires maintainer access to merge changes, and the overhead of rebasing is high.  If you do want to use a feature branch:
```

Comment:
An example of a successful feature branch would be nice to refer to. 

Incase you need one: https://github.com/opensearch-project/opensearch/tree/feature/extensions :)

Reply:
I'm worried about using a specific example since that branch will (soon! SOON! ;) ) get deleted once the feature is launched and I'll have to remember to come back and update/delete it.  WDYT?

Reply:
Makes sense. I agree they will become stale.